README.md

```diff
@@ -1,21 +1,2 @@
-# Hacktoberfest 2018 for pisca
-
-## Sergi, what do you think about Github?
-I love it!
-## Irma, what do you think about CSS?
-a tricky way to make designers feel as IT experts. :D
-
-## Xevi, I guess you know how it works, right?
-sure. still stuck with sth wierd
-
-bailongo, stop living in holidays, please. 
-
-## complains area
-
-### Cinnamon complains
```

Comment:
Good old times when @ramyroquay was obsessed with cinnamon... what are you addicted to now?